### ISSUE TITLE
make manager optional argument in HDF5IO

### ISSUE THREAD

**Issue by neuromusic** [NeurodataWithoutBorders/pynwb#103]
current use
```
from form.backends.hdf5 import HDF5IO
from pynwb import get_build_manager

manager = get_build_manager()
io = HDF5IO(filename, manager, mode='w')
io.write(f)
io.close()
```

this includes two lines that look like boilerplate:

```
from pynwb import get_build_manager

manager = get_build_manager()
```

this should be simplified to 

```
from form.backends.hdf5 import HDF5IO

io = HDF5IO(filename, mode='w')
io.write(f)
io.close()
```

while allowing the user to specify a different build manager if they want it

**Comment by ajtritt** [comment]
resolved in 7bdcd324a5ffde35b50b62886935e2f6694ebe93. 

can close when PR-#98 merged

**Comment by ajtritt** [comment]
should have automatically been closed, but I wrote the wrong thing in my commit message
